iris/iris.go

```diff
@@ -140,3 +150,34 @@ func (c *Client) NewRequestWithContext(ctx context.Context, r *http.Request, met
 
 	return req, nil
 }
+
+func GetErr(res *http.Response) error {
+	if res.StatusCode < 400 {
+		return nil
+	}
+
+	var err siber.JSendError
+	if err := json.NewDecoder(res.Body).Decode(&err); err != nil {
+		panic(err)
+	}
+	return err
+}
+
+func GetResponse(res *http.Response, v interface{}) (interface{}, error) {
```

Comment:
the value you're inflating `v` is already being passed as a reference, why are you returning it